build.sbt

```diff
@@ -5,7 +5,7 @@ version := "1.0"
 lazy val `play-webgoat` = (project in file(".")).enablePlugins(PlayScala)
 
 crossScalaVersions := Seq("2.12.7", "2.11.12")
-scalaVersion := crossScalaVersions.value.head
+scalaVersion := "2.12.8"
```

Comment:
@renatocaval the inconsistency between scalaVersion and crossScalaVersions here is odd and presumably unintended

Reply:
sure, I need to review it. 

We update those projects using TemplateControl (http://github.com/lightbend/templatecontrol) and this particular project is not really aligned and it's causing this kind of problem. 

TemplateControl sees a line with `scalaVersion :=` and decides to replace it with what it has configured. 

I will create an issue on TemplateControl so we don't forget to find a solution for this.

Reply:
@renatocaval 	you can use `tc-skip` on that line to ignore it 

https://github.com/lightbend/templatecontrol/blob/master/src/main/scala/templatecontrol/Task.scala#L99

Reply:
Absolutely, especially because tc-skip was my idea. How could I have forgotten it? ;-)